examples/nodejs-server/README.md

```diff
@@ -1,3 +1,30 @@
-## Node.js Example Project
+## Node.js Example Project with Encryption
 
-See the [guide](https://app-config.netlify.app/guide/node/example.html).
```

Comment:
did you adjust the site and line numbers?

Reply:
Not entirely sure what you mean by that, could you clarify please?

Reply:
Sorry, the docs I mean - it references/includes files from example projects and highlights specific LOCs

Reply:
OH, thanks!

Updated.